ocaml/xapi/records.ml

```diff
@@ -554,6 +554,7 @@ let pool_record rpc session_id pool =
       make_field ~name:"cpu_info" ~get:(fun () -> Record_util.s2sm_to_string "; " (x ()).API.pool_cpu_info) ~get_map:(fun () -> (x ()).API.pool_cpu_info) ();
       make_field ~name:"policy-no-vendor-device" ~get:(fun () -> string_of_bool (x ()).API.pool_policy_no_vendor_device) ~set:(fun s -> Client.Pool.set_policy_no_vendor_device rpc session_id pool (safe_bool_of_string "policy-no-vendor-device" s)) ();
       make_field ~name:"live-patching-disabled"  ~get:(fun () -> string_of_bool (x ()).API.pool_live_patching_disabled) ~set:(fun s -> Client.Pool.set_live_patching_disabled rpc session_id pool (safe_bool_of_string "live-patching-disabled" s)) ();
+      make_field ~name:"uefi-certificates" ~get:(fun () -> (x ()).API.pool_uefi_certificates) ()
```

Comment:
Might want to set this one to `hidden:true`, it'll be giant blob of base64 encoded data. We wouldn't want to show it by default (see how the nvram field is marked)